src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -14096,5 +14086,44 @@ private new struct ResetPoint
             node = this.AddTrailingSkippedSyntax(node, trailingTrash.Node);
             return node;
         }
+
+        private static bool ContainsErrorDiagnostic(GreenNode node)
```

Comment:
copied out from the other PR.  if this goes in first, i'll unify that one on his.

Reply:
The current diff shows this being deleted from above and added here.